bluesky/plans.py

```diff
@@ -1164,7 +1160,8 @@ def monitor_during_wrapper(plan, signals):
     --------
     `bluesky.plans.fly_during_wrapper`
     """
```

Comment:
Agree that my scheme for defaulting name was not configurable enough. Could we keep `sig.name + '-monitor` as a default?